docs/useGetMany.md

```diff
@@ -49,3 +49,42 @@ dataProvider.getMany('tags', { ids: [1, 2, 3, 4, 5, 6, 7] });
 ```
 
 React-admin uses `useGetMany` in [the `<ReferenceField>` component](./ReferenceField.md), to overcome the n+1 problem when using this component in a list. 
+
+## TypeScript
+
+The `useGetMany` hook accepts a generic parameter for the record type:
+
+```tsx
+import { useGetMany, useRecordContext } from 'react-admin';
+
+type Post = {
+    id: number;
+    title: string;
+    tagIds: number[];
+};
+
+type Tag = {
+    id: number;
+    name: string;
+}
+
+const PostTags = () => {
+    const post = useRecordContext<Post>();
+
+    const { data: tags, isLoading, error } = useGetMany<Tag>(
+        'tags',
+        // TypeScript knows that post is of type Post
```

Comment:
I find this comment disturbing. It's drawing my attention to something else than what I'm supposed to learn here. I'd remove it. Same for other locations where you use `useRecordContext` with a type.